### DISCUSSION TITLE
Discrepancy marginaleffects in R and margins command in Stata

### DISCUSSION THREAD

**Discussion by pollyst8** [vincentarelbundock/marginaleffects#1200] [Q&A]
Hello, 

My question refers to the R package #marginaleffects and the Stata command margins, which seem to return different estimates for equivalent functions executed in similar ways. The discrepancy is not too large but is not due to rounding rules. I would like to understand why. 

Using the mtcars data available in base R, 
`car <- lm(mpg ~ wt * drat + am, data=mtcars)`
`library(marginaleffects)`
`predictions(car, newdata=datagrid(wt=seq(1, 5, by=.5), drat=c(2, 5)))`

R retrurns: 

>wt drat Estimate Std. Error      z Pr(>|z|)    S  2.5 % 97.5 % am
 1.0    2    21.12       4.70  4.490   <0.001 17.1 11.901   30.3  0
 1.0    5    41.68       4.16 10.020   <0.001 76.1 33.530   49.8  0
 1.5    2    20.50       3.98  5.156   <0.001 21.9 12.706   28.3  0

Using the same data in Stata, the following command: 
`reg mpg c.wt##c.drat am
margins, at(wt=(1 (.5) 5) drat=(2 5))`

> ------------------------------------------------------------------------------
             |            Delta-method
             |     Margin   Std. Err.      t    P>|t|     [95% Conf. Interval]
-------------+----------------------------------------------------------------
         _at |
          1  |   20.52733   4.781105     4.29   0.000     10.71731    30.33734
          2  |   41.08823   3.768129    10.90   0.000     33.35666    48.81979
          3  |   19.90286   4.069634     4.89   0.000     11.55266    28.25306

Thank you!

**Comment by vincentarelbundock** [comment]
These commands are not equivalent and should not produce the same results. There is an extensive comparison between the two software packages here:

https://marginaleffects.com/vignettes/alternative_software.html

**Reply by pollyst8** [reply]
Thank you, that's very helpful!
